src/lib/util/my_list.ml

```diff
@@ -12,23 +12,10 @@
 (*                                                                        *)
 (*     ---------------------------------------------------------------    *)
 (*                                                                        *)
-(*     The Alt-Ergo theorem prover                                        *)
-(*                                                                        *)
-(*     Sylvain Conchon, Evelyne Contejean, Francois Bobot                 *)
-(*     Mohamed Iguernelala, Stephane Lescuyer, Alain Mebsout              *)
-(*                                                                        *)
-(*     CNRS - INRIA - Universite Paris Sud                                *)
-(*                                                                        *)
-(*     ---------------------------------------------------------------    *)
-(*                                                                        *)
```

Comment:
This needs to stay since we simply rename the file while keeping the existing code.